CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## 1.5.0
 
+  * Deprecate `insertBehind`. This function does not play nicely with merges, I
+    don't have a lot of faith that it works properly without merges, and it is
+    quite slow. ([#35](https://github.com/lspitzner/pqueue/issues/35))
```

Comment:
```suggestion
  * Deprecate `insertBehind`. This function does not play nicely with merges, I
    don't have a lot of faith that it works properly without merges, and it is
    quite slow. ([#35](https://github.com/lspitzner/pqueue/issues/35))
```
IMO changelogs should be a bit more formal (i.e. no "I don't have a lot of faith").

Reply:
I can be more formal. Suggestions?